lenovo/thinkpad/x260/default.nix

```diff
@@ -0,0 +1,20 @@
+{
+  imports = [
+    ../.
+    ../acpi_call.nix
+    ../../../common/cpu/intel
+  ];
+
+  # See https://linrunner.de/en/tlp/docs/tlp-faq.html#battery https://wiki.archlinux.org/index.php/TLP#Btrfs
+  services.tlp.extraConfig = ''
+    START_CHARGE_THRESH_BAT0=75
+    STOP_CHARGE_THRESH_BAT0=80
+    START_CHARGE_THRESH_BAT1=75
+    STOP_CHARGE_THRESH_BAT1=80
+
+    SATA_LINKPWR_ON_BAT=max_performance
+
+    CPU_SCALING_GOVERNOR_ON_BAT=powersave
+    ENERGY_PERF_POLICY_ON_BAT=powersave
+  '';
```

Comment:
I've tested this config on i7-6500U laptop (Purism Librem 13v3) for several days, which is the same CPU generation that X260 comes with (Skylake), but supposedly somewhat more performant. Unfortunately, this config (specifically, `CPU_SCALING_GOVERNOR_ON_BAT=powersave` and `ENERGY_PERF_POLICY_ON_BAT=powersave`) causes severe performance degradation to the point where, e.g. GNOME would not run smoothly anymore.

As of charge thresholds, that seems to be opinionated as well, and I can see other people (myself included)  would like to leverage full battery capacity instead. So maybe let's stick with TLP defaults on that as well.